src/EnlistmentInfo.targets

```diff
@@ -8,10 +8,11 @@
     <DocumentationFile Condition=" !$(IsTestProject) ">$(OutputPath)$(AssemblyName).xml</DocumentationFile>
     <DefineConstants Condition=" '$(TargetFrameworkProfile)' == '' ">$(DefineConstants);DESKTOP</DefineConstants>
   </PropertyGroup>
-
+  
```

Comment:
Let's avoid trailing whitespace, please. :) Format Document ought to fix it (hopefully it won't re-indent all lines though).